test/module-registry.spec.js

```diff
@@ -88,4 +92,52 @@ describe('Module Registry', () => {
     expect(ModuleRegistry.component('GLOBAL_ID')).to.be.undefined;
     expect(ModuleRegistry.invoke('GLOBAL_ID')).to.be.undefined;
   });
+
+  describe('ReactModuleContainerError', () => {
+    let reactModuleContainerErrorCallback;
```

Comment:
I'd make it `const` and call `resetHistory` on `afterEach`

Reply:
Why?

Reply:
I prefer `const`